crates/typed-store/src/rocks/mod.rs

```diff
@@ -207,13 +207,21 @@ impl<K, V> DBMap<K, V> {
         DBBatch::new(&self.rocksdb, &self.db_metrics, &self.write_sample_interval)
     }
 
-    fn cf(&self) -> Arc<rocksdb::BoundColumnFamily<'_>> {
+    pub fn compact_range<J: Serialize>(&self, start: &J, end: &J) -> Result<(), TypedStoreError> {
```

Comment:
this should be a test-only method?

Reply:
You mean put `#[cfg(test)]` in here? Sure, we can do it for now until this finds more use cases